avm/res/container-service/managed-cluster/main.bicep

```diff
@@ -297,6 +297,12 @@ param enableWorkloadIdentity bool = false
 @description('Optional. Whether to enable Azure Defender.')
 param enableAzureDefender bool = false
 
+@description('Optional. Whether to enable Image Cleaner for Kubernetes.')
+param enableImageCleaner bool = false
+
+@description('Optional. The interval in hours Image Cleaner will run. Defaults 120, 5 days.')
+param imageCleanerIntervalHours int = 120
```

Comment:
```suggestion
@description('Optional. The interval in hours Image Cleaner will run. Minimum value is 24 hours and maximum is three months')
param imageCleanerIntervalHours int = 24
```

Reply:
Thanks. I will get this sorted first thing in the morning and then kick off another update and test. 

I have a few more to do soon too. It's becoming easier now. Thanks to all of your help.